icc-x-api/icc-crypto-x-api.ts

```diff
@@ -1392,6 +1409,10 @@ export class IccCryptoXApi {
     return localStorage.getItem(this.keychainLocalStoreIdPrefix + id)
   }
 
+  getKeychainValidityDateInBrowserLocalStorage(id: string): string {
```

Comment:
The setter takes a number but the getter returns a string. Perhaps uniformizing the types will solve the issue with `undefined` value.

Reply:
@cebamps I'll make the change accordingly